script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_3.txt

```diff
@@ -379,48 +379,48 @@ Bad luck is bad luck, but this is just a stroke of incredibly bad luck.
 If this whole ridiculous story is true, then I'm not the only one who's crazy.
 <Page93>
 "...Okay, I'm calm now.
-You have something to say, don't you? Go ahead, say what you want, whether it's a complaint or a grudge."
+You have something to say, don't you? Go ahead and say whatever grudge you have."
```

Comment:
Go ahead and tell me whatever it is you have a grudge about.

Grudge is defined to be 'a feeling of ill will or resentment' when used as a noun.

Reply:
"Go ahead and get whatever complaints or grudges you may have off your chest" ?

>文句でも恨み言でも、思う存分言ってくれ